src/app/about-modal/about-modal.component.less

```diff
@@ -1,6 +1,6 @@
-@import (reference) '../../assets/stylesheets/_base';
-.modal-dialog-f8 { width: 696px; }
-.about-modal-f8 {
+@import (reference) '../../assets/stylesheets/shared/osio.less';
+.io-modal-dialog { width: 696px; }
```

Comment:
The other `io` specific variables all use `io-` currently. This was change was made to match those. If all of the classes should be prefixed with `f8`, I can do that separately.